classes/message_processor.php

```diff
@@ -6,7 +6,7 @@
 use tool_messagebroker\message\immutable_message;
 use tool_messagebroker\receiver\message_receiver;
 use tool_messagebroker\receiver\processing_style;
-use messagebrokerdatastore_standarddb\durable_dao_factory;
+use mbdatastore_standarddb\durable_dao_factory;
```

Comment:
This isn't the right durable_dao_factory that should be used.
It should be `tool_messagebroker\message\durable_dao_factory`

Reply:
Good catch, updated that and retested.